src/chainparamsbase.cpp

```diff
@@ -38,19 +38,19 @@ const CBaseChainParams& BaseParams()
 }
 
 /**
- * Port numbers for incoming Tor connections (8334, 18334, 38334, 18445) have
+ * Port numbers for incoming Tor connections (22557, 44557, 55557, 33557) have
  * been chosen arbitrarily to keep ranges of used ports tight.
  */
 std::unique_ptr<CBaseChainParams> CreateBaseChainParams(const std::string& chain)
 {
     if (chain == CBaseChainParams::MAIN) {
-        return MakeUnique<CBaseChainParams>("", 8332, 8334);
+        return MakeUnique<CBaseChainParams>("", 22555, 22557);
```

Comment:
why are the doors hardcoded? (maybe using constants or macros would be better)

Reply:
It's an odd choice in Bitcoin Core, but typically we don't refactor unnecessarily, to keep it easier to apply patches from upstream.